tests/markdowns/without-annotations/anchors_in_headers.md

```diff
@@ -0,0 +1,9 @@
+<!--
+ - SPDX-FileCopyrightText: 2018-2019 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -->
+
+## <a name="stuff-section"></a> Some stuff
+
+[My link](#stuff-section)
```

Comment:
It seems anchors in headers are no longer ignored, but the validation still doesn't seem to work correctly.

```
$ stack exec xrefcheck -- -r tests/markdowns/without-annotations/anchors_in_headers.md

Configuration file not found, using default config for GitHub repositories

=== Invalid references found ===

  ➥  In file tests/markdowns/without-annotations/anchors_in_headers.md
     bad reference (local) at src:9:1-25:
       - text: "My link"
       - link: 
       - anchor: stuff-section
     
     ⛀  Anchor 'stuff-section' is not present
     
     
Invalid references dumped, 1 in total.
```

Reply:
Nevermind, this is indeed working as expected :D